### DISCUSSION TITLE
429 Too many requests error on node js application

### DISCUSSION THREAD

**Discussion by binilrajan0078** [getsentry/support#51] [Support] [answered]
**Hey!  am a node js developer, so currently am trying to integrate sentry into one of my projects 
but when trying to integrate sentry using the sample code below got an error code of 429 from the sentry side. Is there anything that am missing?**
```
import * as Sentry from "@sentry/node";
import * as Tracing from "@sentry/tracing";

import { config } from "dotenv";

const envFile = process.env.NODE_ENV ? `.env.${process.env.NODE_ENV}` : ".env";
config({ path: envFile });

function initializeAppMonitor(app) {
  if (process.env.NODE_ENV !== "performance") {
    Sentry.init({
      dsn: process.env.SENTRY_DSN,
      environment: process.env.NODE_ENV,
      integrations: [
        new Sentry.Integrations.Http({ tracing: true }),
        new Tracing.Integrations.Postgres(),
        new Tracing.Integrations.Express({ app })
      ],
      tracesSampleRate: 1.0
    });
    app.use(Sentry.Handlers.requestHandler());
    app.use(Sentry.Handlers.tracingHandler());
    app.use(Sentry.Handlers.errorHandler());
  }
}

export default initializeAppMonitor;

```

**Comment by danielkhan** [comment, marked as answer]
This error occurs when a rate limit was reached. Most likely, you have you are over some quota. 
Please review [the docs](https://docs.sentry.io/product/accounts/quotas/) for more information about quota management.
regardless of this, I've created https://github.com/getsentry/sentry/issues/37103 to add actionable information to this kind of error.

**Comment by LeiGao2019** [comment]
@danielkhan , this also brought negative attention and confusion when troubleshooting cross teams. It would be great if Sentry could add an option for it to not appear at all in the console, even the option might block the ability to know if over quota or not.

**Comment by danielkhan** [comment]
I hear you, @LeiGao2019. I did a brief research and it seems as if Chrome would log any `fetch()` call that returns anything `>= 400` to the console. Here is a [Stack Overflow Conversation](https://stackoverflow.com/questions/41515732/hide-401-console-error-in-chrome-dev-tools-getting-401-on-fetch-call). 

Technically, the `429` coming back from the API when a rate limit is exceeded, seems semantically correct. 
The question is if you would not want to know that you are sending data just to be discarded.
Instead of muting this on the backend, would it also be feasible for you to be able to modify a sampling rate setting to stay below the limit?
